stdlib/mmap.pyi

```diff
@@ -34,9 +34,21 @@ class mmap:
     if sys.platform == "win32":
         def __init__(self, fileno: int, length: int, tagname: str | None = ..., access: int = ..., offset: int = ...) -> None: ...
     else:
-        def __init__(
-            self, fileno: int, length: int, flags: int = ..., prot: int = ..., access: int = ..., offset: int = ...
-        ) -> None: ...
+        if sys.version_info >= (3, 13):
+            def __init__(
+                self,
+                fileno: int,
+                length: int,
+                flags: int = ...,
+                prot: int = ...,
+                access: int = ...,
+                offset: int = ...,
+                trackfd: bool = ...,
```

Comment:
According to the docs, `trackfd` is keyword only. Also, we have started to use actual defaults in typeshed, if possible:

```suggestion
                offset: int = ...,
                *,
                trackfd: bool = True,
```

Reply:
Thanks for pointing that. I really forgot the keyword-only part.